### ISSUE TITLE
script not add hdd to rx1214rp

### ISSUE THREAD

**Issue by cyonixx** [007revad/Synology_HDD_db#288]
on my rs4021xs+ your script is not adding the MG09ACA18TE to RX1214rp db file. can you add it please?

../syno_hdd_db.sh 
Synology_HDD_db v3.5.89
RS4021xs+ DSM 7.2.1-69057-3 
StorageManager 1.0.0-0017

rs4021xs+_host_v7 version 8035

Running from: /volume2/BackupAllMCPs/syno_hdd_db.sh

HDD/SSD models found: 2
MG09ACA18TE,0104
MG09ACA18TE,0105

M.2 drive models found: 1
Force MP600,EGFM11.3

M.2 PCIe card models found: 1
M2D20

Expansion Unit models found: 2
RX1214rp
RX1217

Updated MG09ACA18TE in rs4021xs+_host_v7.db
Updated MG09ACA18TE in rx1217rp_v7.db
Updated MG09ACA18TE in rx1217sas_v7.db
Updated MG09ACA18TE in rx1217_v7.db
Updated MG09ACA18TE in rs4021xs+_host_v7.db
Updated MG09ACA18TE in rx1217rp_v7.db
Updated MG09ACA18TE in rx1217sas_v7.db
Updated MG09ACA18TE in rx1217_v7.db
Force MP600 already exists in rs4021xs+_host_v7.db
Force MP600 already exists in rs4021xs+_m2d20_v7.db

M2D20 NVMe already enabled for RS4021xs+

**Comment by 007revad** [comment]
What does the following command return?
```
find /var/lib/disk-compatibility/ -maxdepth 1 -name "rx1214rp*.db"
```

**Comment by 007revad** [comment]
I'll make a debug version of the script so we can see what it's doing wrong.

**Comment by 007revad** [comment]
What's happening is the script seems to only process the last expansion unit in the list.

I can reproduce the problem here so I can fix it a release new version of the script.

**Comment by 007revad** [comment]
Can you try this version:
https://github.com/007revad/Synology_HDD_db/releases/tag/v3.5.90-RC

**Comment by cyonixx** [comment]
great! it works now.

**Comment by 007revad** [comment]
I've just released a final 3.5.90 version. You should update to it so the script can update to, or notify you of, any future newer versions.
https://github.com/007revad/Synology_HDD_db/releases
